docs/using.adoc

```diff
@@ -208,41 +209,24 @@ OpenShift.
 === Using `docker`
 
 ADB provides a full container environment and runs both `docker` and
-`kubernetes`. All standard commands work, for example:
-
-....
-ADB pull centos
- docker run -t -i centos /bin/bash
-....
+`kubernetes`, with most standard commands. 
```

Comment:
@budhrg so should we just keep docker here? this section will become very small and maybe not so helpful. where can I find links to relevant topics or projects that we can send the user to?